content/en/docs/setup/install/istioctl/index.md

```diff
@@ -364,40 +364,89 @@ The `IstioOperator` API defines components as shown in the table below:
+1. [Readiness probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/)
+1. [Replica count](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/)
+1. [`HorizontalPodAutoscaler`](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale/)
+1. [`PodDisruptionBudget`](https://kubernetes.io/docs/concepts/workloads/pods/disruptions/#how-disruption-budgets-work)
+1. [Pod annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
+1. [Service annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
+1. [`ImagePullPolicy`](https://kubernetes.io/docs/concepts/containers/images/)
+1. [Priority class name](https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass)
+1. [Node selector](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector)
+1. [Affinity and anti-affinity](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity)
+1. [Service](https://kubernetes.io/docs/concepts/services-networking/service/)
+1. [Toleration](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/)
+1. [Strategy](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/)
+1. [Env](https://kubernetes.io/docs/tasks/inject-data-application/define-environment-variable-container/)
+1. [Pod security context](https://kubernetes.io/docs/tasks/configure-pod-container/security-context/#set-the-security-context-for-a-pod)
 
-Alternatively, you can disable the telemetry component using a configuration overlay file:
+All of these Kubernetes settings use the Kubernetes API definitions, so [Kubernetes documentation](https://kubernetes.io/docs/concepts/) can be used for reference.
 
-1. Create this file with the name `telemetry_off.yaml` and these contents:
+The following example overlay file adjusts the resources and horizontal pod autoscaling
+settings for Pilot:
 
 {{< text yaml >}}
 apiVersion: install.istio.io/v1alpha1
 kind: IstioOperator
 spec:
   components:
-    telemetry:
-      enabled: false
+    pilot:
+      k8s:
+        resources:
+          requests:
+            cpu: 1000m # override from default 500m
+            memory: 4096Mi # ... default 2048Mi
+        hpaSpec:
+          maxReplicas: 10 # ... default 5
+          minReplicas: 2  # ... default 1
+        nodeSelector:
+          master: "true"
+        tolerations:
+        - key: dedicated
+          operator: Exists
+          effect: NoSchedule
+        - key: CriticalAddonsOnly
+          operator: Exists
 {{< /text >}}
 
-1. Use the `telemetry_off.yaml` overlay file with the `istioctl install` command:
+Use `istioctl install` to apply the modified settings to the cluster:
 
-{{< text bash >}}
-$ istioctl install -f telemetry_off.yaml
+{{< text syntax="bash" repo="operator" >}}
+$ istioctl install -f samples/operator/pilot-k8s.yaml
```

Comment:
The merge conflict is because of a recent fix in the sample yaml files paths. If you fix this, you're version is correct.
```suggestion
$ istioctl install -f operator/samples/pilot-k8s.yaml
```

Reply:
done